doc/Extensions/RRDCached.md

```diff
@@ -11,6 +11,23 @@ To enable this set the following config:
 $config['rrdtool_version'] = 1.5.5;
 ```
 
+### Support matrix
+
+Shared FS: Is a shared filesystem required?
+
+Features: Supported features in the version indicated.
+
+          G = Graphs.
+
+          C = Create RRD Files.
```

Comment:
Put tune and update in here? 

Also the table below could be:
| Version | Shared FS Required | With Shared FS | Without Shared FS| 
| ------- | :-------: | :-------: | -------- |
| **<1.5.5**   | Yes | C,U,G,T  |        |
| **>=1.5.5** | No | C,U,G,T  | C,U  |